core/cloudflow-operator/src/main/resources/application.conf

```diff
@@ -109,6 +109,8 @@ cloudflow {
         # Do not specify max heap, this is calculated by memory-overhead
         java-opts = ""
         java-opts = ${?SPARK_EXECUTOR_JAVA_OPTS}
+
+        persistent-storage = ${cloudflow.platform.deployment.persistent-storage}
```

Comment:
Maybe this is a good time to make this more consistent with flink-runner config, and have:
```hocon
spark-runner {
  driver {
    # contents of spark-runner-driver
  }
  executor {
    # contents of spark-runner-executor
  }
  persistent-storage = ${cloudflow.platform.deployment.persistent-storage}
}
```